run.py

```diff
@@ -920,6 +921,28 @@ def store_cluster_state(ceph_cluster_object, ceph_clusters_file_name):
     log.info("ceph_clusters_file %s", ceph_clusters_file_name)
 
 
+def collect_recipe(ceph_cluster):
+    version_datails = {}
+    client_node = ceph_cluster.get_ceph_objects("client")
+    out, rc = client_node[0].exec_command(
+        sudo=True, cmd="podman --version | awk {'print $3'}"
+    )
+    output = out.read().decode().rstrip()
+    log.info(f"Podman Version {output}")
+    version_datails["PODMAN"] = output
+    out, rc = client_node[0].exec_command(
+        sudo=True, cmd="ceph --version | awk {'print $3'}"
+    )
+    output = out.read().decode().rstrip()
+    log.info(f"ceph Version {output}")
+    version_datails["CEPH"] = output
+    version_detail = open(
+        "/ceph/cephci-jenkins/latest-rhceph-container-info/version_info.json", "w+"
```

Comment:
This works only if the fileshare is mounted. Plus there is high possibility of a race condition. IMO, we should drop this version in the current working directory.